packages/babel-generator/test/fixtures/importAttributesKeyword/attributes-assert-to-default-babel-7/output.js

```diff
@@ -1 +1 @@
-import "a" with type: "json";
\ No newline at end of file
+import "a" with { type: "json" };
```

Comment:
We output the standard syntax on input `import "a" assert { type: "json" };` and options
```json
{
  "BABEL_8_BREAKING": true,
  "plugins": ["deprecatedImportAssert"]
}
```